app/src/components/dialog/EditDialog.tsx

```diff
@@ -76,6 +81,7 @@ export const EditDialog: React.FC<IEditDialogProps> = (props) => {
         }}>
         {(formikProps) => (
           <Dialog
+            fullScreen={fullScreen}
             maxWidth="xl"
```

Comment:
I wonder if there is some way to leverage breakpoints to make the maxWidth dynamic.  In combination with making the width always 100%.  So for example, if the screen is XL, then the dialog is 100% of maxWidth="md" (or something like that).  And as the screen gets smaller, we decrease the maxWidth less and less, until its the same as the screen size (aka: fullscreen)

Reply:
hmm yeah possibly - might be something to discuss in further detail with Jeremy 
For now, I think this is a good fix for our dialog issues and more detailed fixes can be looked into later for smaller viewports :) 
(as per discussion with jer this morning)